docs/changes/0.9.0.rst

```diff
@@ -1,31 +1,40 @@
-Changelog for QCoDeS 0.9.0
-==========================
+Changelog for QCoDeS 0.9.0rc1
+=============================
 
 (NOTE RELEASED YET) The December 2019 release of QCoDeS
```

Comment:
do i understand it correctly that becuase it's an rc we don't remove this "note", right?
```suggestion
The December 2019 release of QCoDeS
```

Reply:
Yes since this goes on the homepage I want to leave this until just before pushing the release to avoid confusion